src/js/mep-feature-universalgoogleanalytics.js

```diff
@@ -8,8 +8,8 @@
 $.extend(mejs.MepDefaults, {
 	googleAnalyticsTitle: '',
 	googleAnalyticsCategory: 'Videos',
-	googleAnalyticsEventPlay: 'Play',
-	googleAnalyticsEventPause: 'Pause',
+	googleAnalyticsEventPlay: 'mejs.play',
```

Comment:
Good catch! this would clearly be a regression. 

Please help with Git/GitHub/PR best practices... I'm a noob here : Should I add a new commit with a fix ? or somehow rewrite my commit in order to exclude this file, and which way would you recommend ?